cli_tools/common/utils/param/populator_test.go

```diff
@@ -456,14 +556,21 @@ func TestPopulator_PopulateMissingParametersPopulatesStorageLocationWithScratchB
 	mockStorageClient := mocks.NewMockStorageClientInterface(mockCtrl)
 	mockStorageClient.EXPECT().GetBucketAttrs("scratchbucket").Return(&storage.BucketAttrs{Location: region}, nil)
 	mockResourceLocationRetriever.EXPECT().GetLargestStorageLocation(region).Return("US")
-
+	mockNetworkResolver := newLenientNetworkResolver(mockCtrl)
 	err := NewPopulator(
+		mockNetworkResolver,
 		mockMetadataGce,
 		mockStorageClient,
 		mockResourceLocationRetriever,
 		mockScratchBucketCreator,
-	).PopulateMissingParameters(&project, "gcloud", &zone, &region, &scratchBucketGcsPath, file, &storageLocation)
+	).PopulateMissingParameters(&project, "gcloud", &zone, &region, &scratchBucketGcsPath, file, &storageLocation, &network, &subnet)
 
 	assert.Nil(t, err)
 	assert.Equal(t, "US", storageLocation)
 }
+
+func newLenientNetworkResolver(ctrl *gomock.Controller) NetworkResolver {
```

Comment:
`lenient` -> `noOp` or similar? 

`lenient` sounds like it still does some resolving but ignores errors. This one, on the other hand, doesn't do anything.

Reply:
good call - updated